lib/LaTeXML/Package/xcolor.sty.ltxml

```diff
@@ -290,7 +290,7 @@ sub DecodeColor {
   my $color_re = qr/($color_expr_re)(($func_expr_re)*)/;
 
   my $color;
-  if ($expression =~ /^$color_re$/) {
+  if ($expression =~ /^$color_re\s*$/) {
```

Comment:
Minor, but the trailing space wasn't recognized and was creating 50+ needless errors. Example from arXiv:1705.04144:
```tex
\documentclass{article}
\usepackage{xcolor}
\begin{document}
\color{rgb, 255:red, 190; green, 190; blue, 190 } Test here.
\end{document}
```